README.md

```diff
@@ -17,6 +17,26 @@ The [contributing documentation](CONTRIBUTING.md) covers licencing and the usual
 
 [Cincinnati][] is configured to track the master branch, so it will automatically react to updates made to this repository.
 
+### Graph-Data Skeleton
```

Comment:
Nit: can we call  this "Cincinnati consumption" or something?  "graph-data" is already clear from the repo this lives in, and I'm not sure how widely folks use "skeleton" to refer to the... bare bones of the structure?

Reply:
Done.